mm/mixtures.py

```diff
@@ -423,7 +424,9 @@ def create_files(self):
         self.write_run_script(self.exe, "accumulate-mixtures.config", "accumulate.sh")
 
     def accumulate(self, task_id):
-        self.run_script(task_id, self.out_log_file[task_id], "./accumulate.sh")
+        self.run_script(
+            task_id, self.out_log_file[task_id], "./accumulate.sh", use_tmp_dir=True
```

Comment:
Other options could be to expose `use_tmp_dir` in the args of the job.

Or (imho good solution) to add a member `self.use_tmp_dir` in the `__init__` that is always (True/False) but that can be overridden in the pipeline for specific jobs.